portfolio/src/main/java/com/google/sps/servlets/LoadCommentServlet.java

```diff
@@ -0,0 +1,50 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.Query;
+import com.google.cloud.datastore.QueryResults;
+import com.google.cloud.datastore.StructuredQuery.OrderBy;
+import com.google.gson.Gson;
+
+
+/** Servlet responsible for listing tasks. */
+@WebServlet("/load-comments")
+public class LoadCommentServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      
+    Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
+    Query<Entity> query =
+        Query.newEntityQueryBuilder().setKind("Comment").setOrderBy(OrderBy.desc("timestamp")).build();
+    QueryResults<Entity> results = datastore.run(query);
+    List<String> comments = new ArrayList<>();
+    while (results.hasNext()) {
+      Entity entity = results.next();
+      comments.add(entity.getString("text"));
+      System.out.println(entity.getString("text"));
```

Comment:
Here and below: `System.out.println` should usually only be used for local debugging, and almost should never be pushed to production. I think what you may want to use instead is logging. See: https://www.geeksforgeeks.org/logging-in-java/#:~:text=is%20named%20getLogManager-,Logger,-Class%0AThe%20logger